hell/adventcalendar/2025/6/index.md

```diff
@@ -1,35 +1,146 @@
 ---
-title: "Accessible by Design: The Role of the lang Attribute"
-author: "Your Name"
-author_bio: "Your short bio"
-date: 2025-12-06
+title: "Accessible by Design: The Role of the 'lang' Attribute"
+author: "Todd Libby"
+author_bio: "Current desert nomad, lobster aficionado, long-time developer, founder of RouteReady, and W3C Invited Expert."
+date: 2025-12-66
```

Comment:
Is this supposed to be `06`?

Reply:
Yes, it is. Thank you for spotting that.